src/styles/blocks/main.scss

```diff
@@ -0,0 +1,40 @@
+.main {
+  padding: 50px 40px;
+  display: grid;
+  grid-template-columns: repeat(1, 200px);
+  justify-content: center;
+  gap: 46px 48px;
+
+  @media (min-width: 488px) {
+    grid-template-columns: repeat(2, 200px);
+  }
+
+  @media (min-width: 768px) {
+    grid-template-columns: repeat(3, 200px);
+  }
+
+  @media (min-width: 1024px) {
+    grid-template-columns: repeat(4, 200px);
+  }
+
+  &__card {
+    box-sizing: border-box;
+    width: 200px;
+    height: 408px;
+    border-radius: 5px;
+    border: 1px solid #f3f3f3;
+    position: relative;
+
+    background-image: url(../images/imac.jpeg);
+    background-size: 160px 134px;
+    background-repeat: no-repeat;
+    background-position-x: 19px;
+    background-position-y: 32px;
+
+    transition-duration: $transition;
```

Comment:
Ensure that the variable `$transition` is defined in your SCSS files. If it's not defined, it will cause a compilation error.